app/components/back-office-app-services/variables.tf

```diff
@@ -269,3 +269,8 @@ variable "back_office_documents_system_topic_name" {
   type        = string
   default     = ""
 }
+
+variable "service_bus_nsip_document_topic_name" {
+  description = "Back Office Service Bus name of topic for NSIP docs"
+  type        = string

```

Comment:
I think you need to give this a default of "" if it's not going to be available in uk-south, or otherwise pass the topic replica to uk-south. This is actually something we haven't done yet, since we're using the DR stuff and I don't know how to do it